### ISSUE TITLE
End with different tone, sound

### ISSUE THREAD

**Issue by rancidfrog** [jithware/brethap#72]
It would be nice to have a tone, sound like a bell or gong for when timer reaches 0.
Currently it just ends and you keep waiting in silence until you realize oh session ended 

Also, having a brief delay when starting to set yourself after you click button to start
For example,
[Breathly](https://github.com/mmazzarolo/breathly-app) has a 3 second delay:
![](https://i.imgur.com/NsJtIJH.png)

But having the delay customizable would be great

**Comment by jithware** [comment]
Currently it is possible to turn on text to speech or vibration to indicate the end of a session (see Duration in [Preferences](https://github.com/jithware/brethap#preferences)). A new tone can be added to support this feature. Please provide one or two audio files as documented in https://github.com/jithware/brethap/issues/57#issuecomment-1238629942. Also include a README.md file in the same directory that describes the source of the audio file(s) and/or method of creation if it is an original work.

Once the above is complete, I'll look into adding a delay.

**Comment by jithware** [comment]
Added instructions for audio file inclusion see: https://github.com/jithware/brethap/discussions/76
